pkg/alertmanager/amcfg.go

```diff
@@ -980,6 +984,29 @@ func (cb *ConfigBuilder) convertSlackConfig(ctx context.Context, in monitoringv1
 	return out, nil
 }
 
+func (cb *ConfigBuilder) convertSlackFields(in []monitoringv1alpha1.SlackField) ([]slackField, error) {
```

Comment:
this seems to be unrelated to the mattermost integration, is better for review to no change this